tools/chromium_src/lit_mangler/__snapshots__/mangle.test.ts.snap

```diff
@@ -0,0 +1,25 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`mangled files should have up to date snapshots ./chromium_src/chrome/browser/resources/extensions/detail_view.mangle.html.ts should match snapshot 1`] = `
+"===================================================================
+--- ../chrome/browser/resources/extensions/detail_view.html.ts
++++ ../out/Component/gen/chrome/browser/resources/extensions/preprocessed/detail_view.html.ts
```

Comment:
don't think this will work. we have a lot of other bulid configurations.

Reply:
Updated it to skip the outdirectory in the path - it will now use `gen/....`

Reply:
the snapshot still says `../out/Component/gen`. Assuming you just didn't push the fixed version.

Reply:
Yeah didn't push the fixes, as I got halfway through running from the out directory and realised I'd need to bundle the files